### PR TITLE
feat: Update redirect page styles

### PR DESCRIPTION
**Pull request by andre-code**
PR to apply new design for redirect page.

redirect page

/deploy

### CODE REVIEW

**Review by leafty — COMMENTED** [review]
Imho, we should redirect immediately, removing the need for this page altogether.

**Comment by andre-code** [comment]
Note:
We will wait to determine if the redirect is still necessary. @leafty will work on removing that page in renku-gateway first.

File: client/public/static/public/theme.css [hunk 1]
```diff
@@ -50,43 +50,129 @@ body {
...
 }
 
 .content {
-  background-position: center;
-  background-size: cover;
-  /* calc 100vh - footer height - header height */
   height: calc(100vh - 75px - 70px);
-  background-image: url("./img/404@0.25x.png");
-  color: #d26a98;
+  color: #000;
```
Comment (leafty) [inline client/public/static/public/theme.css:67]: nitpick:

```suggestion
  color: black;
```

File: client/public/static/public/theme.css [hunk 2]
```diff
@@ -50,43 +50,129 @@ body {
...
+}
+
+.content {
+  background-image: url("./img/redirect/frog/X-Large_5.png");
+  background-position: center;
+  background-size: inherit;
+  background-repeat: no-repeat;
+  overflow: auto;
+  width: 100%;
+  animation: changeBackgroundXL 5s step-end;
```
Comment (leafty) [inline client/public/static/public/theme.css:152]: The animation stops after 5 seconds, but the page may be displayed for longer:

```suggestion
  animation: changeBackgroundXL 5s step-end infinite;
```

**Review by leafty — DISMISSED** [review]
LGTM, some comments here:

**Review by leafty — APPROVED** [review]

**Pull request merged**